Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -196,6 +198,19 @@ public bool StartServer()
             if (runInBackground)
                 Application.runInBackground = true;
 
+            // set a fixed tick rate instead of updating as often as possible
+            // * if not in Editor (it doesn't work in the Editor)
+            // * if not in Host mode
+            // => set both Update and FixedUpdate tick rate.
+#if !UNITY_EDITOR
+            if (!NetworkClient.active)
+            {
+                Application.targetFrameRate = serverTickRate; // Update
+                Time.fixedDeltaTime = 1f / serverTickRate; // FixedUpdate
```

Comment:
I don't think we should touch the fixedDeltaTime.

Users can already configure this in their preferences and we should respect user preferences because this normally affects physics.

Reply:
On the other hand, a comprehensive networking library will want to tie physics to network tick rate. I think it makes some sense.